sphinx_design/_compat.py

```diff
@@ -9,3 +10,10 @@ def findall(node: Element) -> Callable[..., Iterable[Element]]:
     # findall replaces traverse in docutils v0.18
     # note a difference is that findall is an iterator
     return getattr(node, "findall", node.traverse)
+
+
+def read_text(module: resources.Package, filename: str) -> str:
+    has_files = hasattr(resources, "files")
+    if has_files:
+        return resources.files(module).joinpath(filename).read_text()
```

Comment:
Seems cleaner than assigning a one-time-use variable.

```suggestion
    if hasattr(resources, "files"):
        return resources.files(module).joinpath(filename).read_text()
```